_posts/2020-04-11-the-definitive-guide.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Presto: The Definitive Guide"
+author: Matt Fuller, Manfred Moser and Martin Traverso
+excerpt_separator: <!--more-->
+---
+
+Nearly two years ago Matt and Martin got the ball rolling on getting a book
+about Presto happening. A thriving project and community like everyone around
+Dain, David and Martin, the founders and creators of Presto, just needs a book.
+Even in this digital age of online documentation, communities on chat and other
+platforms, and videos everywhere, there is great value in a well structured and
+written book.
+
+Today, we are happy to announce that our book __Presto: The Definitive Guide__,
+the first book about Presto, is finally available for you all to get, read and
+hopefully learn from.
+
+<!--more-->
+
+<img src="../../../../assets/ptdg-cover.jpg" align="right" width="400">
+
+With the help of O'Reilly, the book is now available in digital form, and paper
+copies are just around the corner as well. You can find more information about
+the book on [our permanent page about
+it](../../../../presto-the-definitive-guide.html).
+
+It is based on the very recent 330 release of Presto and broken up into three
```

Comment:
Perhaps mention somewhere that although based on 330, it's not really version-specific. Not sure how we'd want to phrase that.

Reply:
Fixed